Exiled.Events/Patches/Events/Player/InsertingGeneratorTablet.cs

```diff
@@ -34,6 +34,38 @@ private static bool Prefix(Generator079 __instance, GameObject person, PlayerInt
                 {
                     case PlayerInteract.Generator079Operations.Door:
                         bool isAllowed = true;
+                        if (!__instance.isDoorUnlocked)
+                        {
+                            isAllowed = person.GetComponent<ServerRoles>().BypassMode;
```

Comment:
You wouldn't want to use a GetComponent<> here, simply use Player.Get(person).Bypass or Player.Get(person).ReferenceHub.serverRoles